ts/Dashboard/Component/Component.ts

```diff
@@ -322,6 +324,14 @@ abstract class Component<TEventObject extends Component.EventTypes = Component.E
     }
 
     public setStore(store: Component.StoreTypes | undefined): this {
+        // Clean up old event listeners
+        while (this.tableEvents.length) {
```

Comment:
I would replace it with:

```
const tableEventsLength =  this.tableEvents.length;
while (tableEventsLength) {
```
As a result, the length will be calculated only one time, instead of each time when run a loop.

Reply:
It needs to be calculated every time, otherwise it will keep on going ♾️

Reply:
Ok!